### PR TITLE
Convert to blivet-2.0 API.

### PR DESCRIPTION
**Pull request by dwlehman**

### CODE REVIEW

File: pyanaconda/kickstart.py [hunk 1]
```diff
@@ -1516,12 +1516,12 @@ def execute(self, storage, ksdata, instClass):
...
+        kwargs["fmt"] = get_format(ty,
            label=self.label,
            fsprofile=self.fsprofile,
            mountpoint=self.mountpoint,
            mountopts=self.fsopts,
-           createOptions=self.mkfsopts)
+           create_options=self.mkfsopts)
         if not kwargs["fmt"].type:
             raise KickstartParseError(formatErrorMsg(self.lineno,
                     msg=_("The \"%s\" file system type is not supported.") % ty))
```
Comment (vojtechtrefny) [inline pyanaconda/kickstart.py]: `kwargs["memberDevices"]` and `kwargs["totalDevices"]` on lines 1532 and 1533 should be `kwargs["member_devices"]` and `kwargs["total_devices"]` (GitHub won't allow me to add a comment to unchanged lines...)

File: pyanaconda/image.py [hunk 2]
```diff
@@ -168,7 +169,8 @@ def opticalInstallMedia(devicetree):
 # Return a list of Device instances that may have HDISO install media
 # somewhere.  Candidate devices are simply any that we can mount.
 def potentialHdisoSources(devicetree):
-    return filter(lambda d: d.format.exists and d.format.mountable, devicetree.getDevicesByType("partition"))
+    return filter(lambda d: d.format.exists and d.format.mountable,
+                  (d for d in devicetree.devices if d.type == "partition"))
```
Comment (vpodzime) [inline pyanaconda/image.py]: The ``filter`` call and the ``lambda`` can be dropped here with all moved to the generator conditions.

File: tests/pylint/runpylint.py [hunk 3]
```diff
@@ -14,7 +14,8 @@ def __init__(self):
 
                                 # XXX: These are temporary until dogtail and koji have python3 versions.
                                 FalsePositive(r"^E0401.*: Unable to import 'dogtail.*'$"),
-                                FalsePositive(r"^E0401.*: Unable to import 'koji'$")
+                                FalsePositive(r"^E0401.*: Unable to import 'koji'$"),
+                                FalsePositive(r"^W9930.*:")
```
Comment (vpodzime) [inline tests/pylint/runpylint.py]: A comment here about the code would be nice.

**Comment by vpodzime** [comment]
Looks good to me otherwise (as much as I can see the real changes here).

File: tests/pylint/runpylint.py (hunk 3 above)
Comment (dwlehman) [inline tests/pylint/runpylint.py]: I didn't mean to include that change at all -- I'll remove it.

File: pyanaconda/kickstart.py (hunk 1 above)
Comment (dwlehman) [inline pyanaconda/kickstart.py]: Good catch -- thanks.

**Comment by vojtechtrefny** [comment]
Found few more "old" API stuff:

`pyanaconda/bootloader.py:390`: `device.metadataVersion` -> `device.metadata_version`
`pyanaconda/kickstart.py:931`: `vg.peSize` -> `vg.pe_size`
`pyanaconda/ui/gui/spokes/custom.py:813`: `self._storage_playground.resizeDevice` -> `self._storage_playground.resize_device`
`pyanaconda/ui/gui/spokes/custom.py:825`: `use_dev.targetSize` -> `use_dev.target_size`
`pyanaconda/ui/gui/spokes/storage.py:760`: `device.format.hasKey` -> `device.format.has_key`

File: pyanaconda/bootloader.py [hunk 4]
```diff
@@ -441,24 +441,24 @@ def _is_valid_size(self, device, desc=""):
         ret = True
         msg = None
         errors = []
-        if device.format.minSize and device.format.maxSize:
+        if device.format.min_size and device.format.maxize:
```
Comment (snbueno) [inline pyanaconda/bootloader.py]: device.format.maxize should be device.format.max_size

**Comment by dwlehman** [comment]
Cherry-picked and pushed as 687d7164.

**Pull request closed without merging**